__tests__/api/league/route.js

```diff
@@ -55,26 +71,14 @@ afterEach(() => {
 });
 
 describe("POST (unit tests)", () => {
-    it("should return a 403 when auth token does not have exact right userId claim", async () => {
-        // Arrange
-        testAuthData.sub = "123googleTestId";
-        const request = {
-            json: async () => { return {
-                token: "testToken"
-            } }
-        };
-
-        // Act
-        const response = await POST(request);
-
-        // Assert
-        expect(response).not.toBeNull();
-        expect(response.status).toEqual(403);
-    });
```

Comment:
Updating the API would again allow us to reinstate this test.

Reply:
Pushed the update up :D